src/components/MovieInfo.tsx

```diff
@@ -1,14 +1,47 @@
 import React from "react";
+import { BiCameraMovie } from "react-icons/bi";
+import { PiTelevisionSimple } from "react-icons/pi";
+
+export type MovieInfoProps = {
+  data: DataChildren[];
+};
+
+export type DataChildren = {
+  Title: string;
+  Year: string;
+  imdbID: string;
+  Type: string;
+  Poster: string;
+};
+
+const MovieInfo: React.FC<MovieInfoProps> = ({ data }) => {
 
-const Card = () => {
   return (
-    <>
-      <li>
-        <img src="" alt="" />
-        <div></div>
-      </li>
-    </>
+    <div className="container">
+      <div className="row justify-content-between align-items-center">
+        {data.map((item, i) => {
+          const { Title, Year, Type, Poster } = item
+          const icon = Type === 'movie' ? <BiCameraMovie /> : <PiTelevisionSimple />
```

Comment:
Let's go ahead and implement this small destructuring within the map params, get rid of the next two lines since we are not longer using them, code should stay like this now: 
```suggestion
{data.map(({ Title, Year, Type, Poster }, i) => {
```